test/cpp_tests_list_IECPU.txt

```diff
@@ -7,85 +7,32 @@
 ####  EXCLUDED GTEST PATTERNS ####
 [EXCLUDED]
 
-BackendManager.* # backend tests don't apply for IE
-NGraphExecTest.Axpy
 NGraphExecTest.Axpy8bit
 NGraphExecTest.MixedTensors
-NGraphExecTest.FindNumberOfNodesUtil1
-NGraphExecTest.FindNumberOfNodesUtil2
-TFExec.SingleGraphOn2Threads
-TFExec.axpy
-tf_exec.BatchMatMul
 EncapsulateOp.GetNgExecutable
 EncapsulateOp.AllocateNGInputTensors
 EncapsulateOp.AllocateNGOutputTensors
+BackendManager.SetBackend
+BackendManager.GetCurrentlySetBackendName
+BackendManager.CanCreateBackend
 OpByOpCapability.Backend
-MathOps.Sum
-MathOpsSumFixture.FullSet
-MathOps.Mean
-MathOps.ArgMaxNeg
-MathOps.ArgMaxPos
-MathOps.ArgMinNeg
-MathOps.ArgMinPos
-MathOps.Atan2
-MathOps.BatchMatMul2D
-MathOps.BatchMatMul2DAdjX
-MathOps.BatchMatMul2DAdjY
-MathOps.BatchMatMul2DAdjXY
-MathOps.BatchMatMul3D
-MathOps.BatchMatMul3DAdjX
-MathOps.BatchMatMul3DAdjY
-MathOps.BatchMatMul3DAdjXY
-MathOps.BatchMatMul4D
-MathOps.BatchMatMul4DAdjX
-MathOps.BatchMatMul4DAdjY
-MathOps.BatchMatMul4DAdjXY
-MathOps.BatchMatMulV22D
-MathOps.BatchMatMulV22DAdjX
-MathOps.BatchMatMulV22DAdjY
-MathOps.BatchMatMulV22DAdjXY
-MathOps.BatchMatMulV23D
-MathOps.BatchMatMulV23DAdjX
-MathOps.BatchMatMulV23DAdjY
-MathOps.BatchMatMulV23DAdjXY
-MathOps.BatchMatMulV24D
-MathOps.BatchMatMulV24DAdjX
-MathOps.BatchMatMulV24DAdjY
-MathOps.BatchMatMulV24DAdjXY
-MathOps.Ceil
-MathOps.FloorDiv
-MathOps.FloorDivInt
-MathOps.FloorDivBroadcasting
-MathOps.FloorDivNegInt
-MathOps.FloorDivNegFloat
-MathOps.IsFinite
-MathOps.LogicalOr
-MathOps.LogicalNot
-MathOps.Pow0D1D
-MathOps.SqueezeNoAttributes
-MathOps.UnsortedSegmentSum
-MathOps.UnsortedSegmentSumIdxRange
-MathOps.UnsortedSegmentSumMissingIndices
-MathOps.UnsortedSegmentSumSingleIndex
-MathOps.UnsortedSegmentSumTwoDims
-MathOps.NotEqual
-NNOps.L2Loss
-NNOps.LogSoftmax
-NNOps.SoftmaxZeroDimTest1
-NNOps.SoftmaxZeroDimTest2
-ArrayOps.Fill
-ArrayOps.GatherNd2D
-ArrayOps.GatherNd3D
-ArrayOps.Rank
-ArrayOps.ScatterNd1D
-ArrayOps.ScatterNdRepeatIndices
-ArrayOps.ScatterNdComplex
-ArrayOps.ScatterNd3D
+MathOps.Cumsum
+MathOps.SquaredDifference
+MathOps.SquaredDifferenceBroadcasting
+MathOps.Mod
+NNOps.Conv2DBackpropInputNCHWSame
```

Comment:
So, these NNOps.conv* tests were passing earlier. Are these failing now due to const folding?

Reply:
Yes, but what's odd is that the failure is not seen when these tests are run individually.

Reply:
If that is reproducible, we should investigate (now or later).